core/src/function.rs

```diff
@@ -117,15 +117,13 @@ impl Function {
 
     fn index_cflow_graph(
         g: ControlFlowGraph,
-        entry: Option<u64>,
+        entry: u64,
     ) -> (BTreeMap<u64, Vec<MnemonicOrError>>, HashMap<u64, Vec<(Rvalue, Guard)>>, HashMap<u64, Vec<(Rvalue, Guard)>>) {
         let mut mnemonics = BTreeMap::new();
         let mut by_source = HashMap::<u64, Vec<(Rvalue, Guard)>>::new();
         let mut by_destination = HashMap::<u64, Vec<(Rvalue, Guard)>>::new();
 
-        if let Some(entry) = entry {
-            by_destination.insert(entry, vec![(Rvalue::Undefined, Guard::always())]);
-        }
+        by_destination.insert(entry, vec![(Rvalue::Undefined, Guard::always())]);
```

Comment:
Similarly, because we must create a function with a start, this no longer makes sense to be optional. I've updated the affected tests accordingly: https://github.com/das-labor/panopticon/blob/5d623578191807b7a640f351cf3fead1faf1278f/core/src/function.rs#L734